components/skeleton/demo/skeleton-host.js

```diff
@@ -0,0 +1,38 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { skeletonStyles } from '../skeleton-styles.js';
+
+class SkeletonHost extends LitElement {
```

Comment:
This is just a demo component showing how other custom components could support the `skeleton` property and then internally use combinations of the `skeleton` attribute on the host and the `skeletize` class to style their own internal content.

e.g. here the component has a list, but as it might not know how long this list will be until loaded it just renders the list at a fixed height. (This example is taken directly from the right hand panel summarizers that we have in FACE, where we only want to render a single block to represent the potential content - other scenarios might want to represent a 'pseudo list' in which case the approaches shown by Maya/Chris might be more appropriate).

Reply:
Could probably also be exposed as a simple `SkeletonMixin` that components could extend -- with it they'd get the `skeleton` property (ensuring naming consistency across components) and it would load the `skeletonStyles` automatically enabling them to use the CSS class.

Reply:
I like the idea of a mixin because it can help to standardize when we want to create more complex skeleton components.

Reply:
I agree mixins are super handy for standardizing implementations of features for components like this. My concern is consumers of the mixin overriding property/function behaviour from it or utilizing properties and functions in odd ways.

The new members of Team Phoenix who haven't worked with polymer or lit before have all expressed frustration with the "opaqueness" of mixins. Mainly not being able to figure out where certain properties or functions come from when they're referenced or overridden in mixin consumers. That's the source of my concerns, otherwise a skeleton mixin would be a huge boon for this feature.

Reply:
I sort of prefer this not to be a mixin. These don't need access to lifecycle lit so having a mixin might be over kill.

Reply:
It doesn't need access to lifecycle methods... _yet_. :)

Without the mixin, components that want to be skeleton-able would need to import the styles themselves and add their own `skeleton` property. Not a big deal... but having the mixin also doesn't prevent them from doing those two things manually either.